### PR TITLE
feat(hpc-vmware-managed-vcd): init hpc VCD react µapp

### PR DESCRIPTION
**Pull request by darsene**
ref: MANAGER-14419

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/hpc-vcd-beta`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-14419
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

New µapp for HPC VMware Cloud Director offer

### CODE REVIEW

**Review by rjamet-ovh — APPROVED** [review]

**Review by vovh — APPROVED** [review]

**Comment by darsene** [comment]
> I think we should fix the tests that are not passing, if i remember well you just have to rename the files to avoid having them running with the wrong test suit

@vovh it's done. script `test:e2e:ci` has been renamed to `test:e2e:cii` temporally

File: packages/manager/apps/hpc-vmware-managed-vcd/mocks/example/project-uuids.json
Comment (anooparveti) [inline packages/manager/apps/hpc-vmware-managed-vcd/mocks/example/project-uuids.json:1]: **question:** Do we need this file?

File: packages/manager/apps/hpc-vmware-managed-vcd/mocks/example/project-uuids.ts
Comment (anooparveti) [inline packages/manager/apps/hpc-vmware-managed-vcd/mocks/example/project-uuids.ts:1]: **question:** Do we need this file?

File: packages/manager/apps/hpc-vmware-managed-vcd/mocks/index.ts
Comment (anooparveti) [inline packages/manager/apps/hpc-vmware-managed-vcd/mocks/index.ts:1]: **question:** Do we need this file?

File: packages/manager/apps/hpc-vmware-managed-vcd/package.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@ovh-ux/manager-hpc-vmware-managed-vcd-app",
+  "version": "0.0.0",
+  "private": true,
+  "description": "New managed VMware cloud director offers",
```
Comment (anooparveti) [inline packages/manager/apps/hpc-vmware-managed-vcd/package.json:5]:
```suggestion
  "description": "New managed VMware Cloud Director offers",
```

File: packages/manager/apps/hpc-vmware-managed-vcd/src/data/hooks/pci/useProject.tsx
Comment (anooparveti) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/data/hooks/pci/useProject.tsx:1]: Looks like an error on generator. Can you remove this file?

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/hpc-vmware-managed-vcd/mocks/example/project-uuids.json
Comment (darsene) [reply packages/manager/apps/hpc-vmware-managed-vcd/mocks/example/project-uuids.json:1]: not anymore as the API will change. I removed all the corresponding files

File: packages/manager/apps/hpc-vmware-managed-vcd/package.json (hunk 1 above)
Comment (darsene) [reply packages/manager/apps/hpc-vmware-managed-vcd/package.json]: Fixed

File: packages/manager/apps/hpc-vmware-managed-vcd/src/data/hooks/pci/useProject.tsx
Comment (darsene) [reply packages/manager/apps/hpc-vmware-managed-vcd/src/data/hooks/pci/useProject.tsx:1]: File removed

**Review by anooparveti — APPROVED** [review]

**Pull request merged**